.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
The gitignore should be part of the `crate-template`. Can you please also submit a PR in `crate-template` to add it.